packages/syft/src/syft/service/queue/zmq_queue.py

```diff
@@ -378,16 +387,21 @@ def update_consumer_state_for_worker(
             )
             return
 
-        res = self.worker_stash.update_consumer_state(
-            credentials=self.worker_stash.partition.root_verify_key,
-            worker_uid=syft_worker_id,
-            consumer_state=consumer_state,
-        )
-        if res.is_err():
+        try:
```

Comment:
Just some extra try catching if something fails during update, it doesn't kill the consumer.